pkg/controllers/provisioningv2/rke2/provisioninglog/provisioninglog.go

```diff
@@ -97,6 +97,10 @@ func (h *handler) recordMessage(provCluster *provv1.Cluster, cm *corev1.ConfigMa
 		return cm, nil
 	}
 
+	if strings.Contains(msg, "the object has been modified; please apply your changes to the latest version and try again") {
+		msg = fmt.Sprintf("%s: this error may resolve itself, and unless consistently repeated, does not represent a critical error in the provisioning process", msg)
```

Comment:
It's difficult to both quantify and tell if this message is being "consistently repeated". Perhaps we can just state that it is a transient error, or format it like `Transient error encountered: %s`

Reply:
Sure. Wording this scenario was a bit difficult, but I think that wording sounds a lot better